data/remnant missions.txt

```diff
@@ -57,8 +55,14 @@ mission "First Contact: Remnant"
 				decline
 	on accept
 		"reputation: Remnant" <?= -10
+		log "Refused to give a blood sample to a group of strange-looking humans that live south of the Core. Upon takeoff, their ships attacked."
+		log "Factions" "Remnant" `A group of unusual-looking humans that live in a secluded section of the galaxy. They are openly hostile to those who refuse to submit blood samples.`
```

Comment:
It may seem a bit strange for the player to have labeled planets/systems, but no faction log for this group, while every other faction - hostile or not - has a faction log (automata excluded, I think). With that in mind, I wanted to offer some kind of faction log to the player. A single sentence indeed seemed too short, and there isn't much else the player knows about the Remnant to include (including the motivation behind the the blood test), if the player runs.

I'm open to suggestions for edits, and if you've got a particular design in mind, I welcome the touchup merge commit 👍  After all, you have a better idea of what's next for a player who has a hostile rep with the Remnant than we do 👼